### ISSUE TITLE
Enhanced SimpleXML benchmarks

### ISSUE THREAD

**Issue by kanevbg** [rusoft/php-simple-benchmark-script#19]
Continuation of #16 and thanks for adding :)

We should add operations with the extension else it is not really useful in practice.

Later I will comment suggested operations to do.

**Comment by sergey-dryabzhinsky** [comment]
Maybe we need to parse something big and complex?
Like unzipped OpenDocument/OpenXML xml file with alot of styles and etc.
Something free-to-distribute.
And try to search something, summarize and calculate something simple.

**Comment by kanevbg** [comment]
Yes, these basic operations seems great. Random data generation tool can be used to generate XML string/file for benchmarking purposes, eg I generated example file with that web tool: https://onlinerandomtools.com/generate-random-xml
[output-onlinerandomtools.txt](https://github.com/rusoft/php-simple-benchmark-script/files/8619267/output-onlinerandomtools.txt)
